net/openssh/Makefile

```diff
@@ -210,13 +210,17 @@ define Build/Compile
 		all install
 endef
 
+define Package/openssh-sysdir/install
```

Comment:
I can not see this package in this Makefile. Maybe, you forgot to add something more to your commit. 
If the package would not be introduced, you can keep to the previous solution and you just need to change chmod, right? :)